utils/utils.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"math/rand"
+	"strings"
+
+	client "github.com/fybrik/datacatalog-go-client"
+	models "github.com/fybrik/datacatalog-go-models"
+)
+
+func AppendStrings(a string, b string) string {
+	if strings.Contains(b, ".") {
+		return a + ".\"" + b + "\""
+	} else {
+		return a + "." + b
+	}
+}
+
+// If the tag is in the Fybrik tag category (e.g. "Fybrik.PII"), remove the
+// "Fybrik." prefix. Otherwise, do nothing
+func StripTag(tag string) string {
+	return strings.TrimPrefix(tag, "Fybrik.")
+}
+
+func UpdateCustomProperty(customProperties map[string]interface{}, orig map[string]interface{}, key string, value *string) {
+	// update customProperties only if there is a new value
+	if value != nil && *value != "" {
+		customProperties[key] = *value
+		return
+	}
+
+	// if there is no new value, revert to original value
+	if v, ok := orig[key]; ok && v != "" {
+		customProperties[key] = v
+	}
+}
+
+const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
a small improvement
```goLang
var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")

func randSeq(n int) string {
    b := make([]rune, n)
    for i := range b {
        b[i] = letters[rand.Intn(len(letters))]
    }
    return string(b)
}
```
plus we need to init the `rand.Seed(time.Now().UnixNano())`  (this can be done in the `init` function )

Reply:
updated RandSeq to use rune.
lint forced me to move from math/rand to cyprto/rand
crypto/rand does not have a seed function